app/views/footprints/new.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="jumbotron text-center">
+  <h5 class="display-4">
+    Calculate Your Car's Footprint for a Month</h5>
+</div>
+
+<div class='container-lg text-center'>
+  <% if @cars.empty? %>
+    <section class='no-cars-alert'>
+      <div class="alert alert-dismissible alert-primary">
+        <strong>Oh snap!</strong> You haven't added any cars yet. <a href='/cars/new' class="alert-link">Add a car</a> to calculate your footprint.
+      </div>
+    </section> 
+  <% end%>
+</div>
+
+<% unless @cars.empty? %>
+  <section class='calculate-footprint'>
+    <div class="form-group container-lg text-center">
+      <%= form_tag '/carbon_calculator', method: :post do %>
+        <div class='form-select form-select-lg mb-3 p-2'>
+          <%= select_tag(:car, options_for_select(@cars.map {|car| "#{car.year} #{car.make} #{car.model}"}, '1')) %>
```

Comment:
I actually just noticed this - I think in the backend we're expecting the car id in the mutation to create a new footprint, so we're going to need to get that into this form, which means we'll probably need it in the car facade and poro too (we just need to add it to the query)